sample-android/src/main/java/io/apptik/jus/samples/MyJus.java

```diff
@@ -58,6 +59,18 @@ public JsonArray convert(NetworkResponse value) throws IOException {
                                 }
                             };
                         }
+                        if (hasAnnotatedTag(annotations, Instructables.REQ_INFO)) {
+                            return new Converter<NetworkResponse, Object>() {
+                                JJsonObjectResponseConverter baseConv = new
+                                        JJsonObjectResponseConverter();
+
+                                @Override
+                                public JsonObject convert(NetworkResponse value) throws
+                                        IOException {
+                                    return baseConv.convert(value);
```

Comment:
Override is useless here in the case above we want to show how response can ve converted uing annotations here we just need plain JJsonObjectResponseConverter